src/task/script.js

```diff
@@ -389,20 +393,32 @@ function clearUserNotFound() {
 function createSuggestionsList(matches) {
   const listItems = document.getElementById('list-items');
   if (matches.length) {
-    matches.map(({ username }) => {
+    assigneeAvatarLoader.style.display = 'initial';
+    matches.map(({ username, picture = {} }) => {
+      const defaultImagePath = '../images/avatar.png';
```

Comment:
This should move to a separate constant file to save the file name.

Reply:
I have created a constant.js file and added this path, now as I have to use that constant.js file in this script.js file, I have to use modules, as now the script.js file is importing constants.js file, it also became module type, and now all the functions which are in the script.js file is not globally accessible, and that's why I am getting reference error on all the functions which are attached in tasks.html file, there is one solution to this issue is to attach all script.js file functions to a window object, but I am not sure this is the right way to go. 
Can u suggest what should be done in this case @Neha